great_expectations/rule_based_profiler/parameter_builder/unexpected_count_statistics_multi_batch_parameter_builder.py

```diff
@@ -127,7 +127,24 @@ def quantile_statistic_interpolation_method(self) -> str:
     def round_decimals(self) -> Optional[Union[str, int]]:
         return self._round_decimals
 
-    def _build_parameters(  # PLR0915
+    def _standardize_mostly(self, mostly: np.float64) -> np.float64:
+        """
+        Standardizes "mostly" value to:
+            1.0 if >= 1.0
+            0.99 if >= 0.99
+            0.975 if >= 0.975 and < 0.99
+        Otherwise, returns the original value.
+        """
+        if mostly >= 1.0:  # noqa:  PLR0915
+            return 1.0
+        elif mostly >= 0.99:  # noqa:  PLR0915
```

Comment:
@TrangPham This does not need `elif` because each clause has a `return`.  Thanks.